extensions/amp-tiktok/validator-amp-tiktok.protoascii

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-tiktok
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-tiktok"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-tiktok>
+  html_format: AMP
+  tag_name: "AMP-TIKTOK"
+  spec_name: "AMP-TIKTOK"
+  requires_extension: "amp-tiktok"
+  attr_lists: "extended-amp-global"
+  attrs: {
+    name: "data-src";
+    value_regex: "\d";
```

Comment:
It seems this should have rather been `\d+`. See https://github.com/ampproject/amphtml/pull/35329.

Reply:
Oh, what's more is that it should apparently allow a URL as well, as this is supposed to be valid:

https://github.com/ampproject/amphtml/blob/f0a90fae17c7224f83f40dddb6583aef4e7d96d1/extensions/amp-tiktok/0.1/test/validator-amp-tiktok.html#L40-L45

Reply:
@westonruter could you open an issue instead of commenting on a merged PR?

Reply:
I opened this PR to fix: https://github.com/ampproject/amphtml/pull/35329.

I just commented on this PR for visibility.

Reply:
Thank you for opening the PR to fix. I still believe an issue would be more informative than effectively re-opening a PR.